src/createServer.js

```diff
@@ -1,11 +1,23 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const cors = require('cors');
+const { usersRouter } = require('./routes/users.js');
+const { expensesRouter } = require('./routes/expenses.js');
+const { expensesService } = require('./services/expenses.js');
+const { usersService } = require('./services/users.js');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  expensesService.resetData();
+  usersService.resetData();
+
+  app.use(cors());
+  app.use('/users', express.json(), usersRouter);
+  app.use('/expenses', express.json(), expensesRouter);
```

Comment:
You can call this only once, to make your code a bit more DRY :)
```suggestion
  app.use(cors());
  app.use(express.json());
  app.use('/users', usersRouter);
  app.use('/expenses', expensesRouter);
```